weather_app/management/commands/load_files_data.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
```

Comment:
wrong import error use pylint to identify and isort for correcting the order of imports